devel/fzy/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        jhawthorn fzy 0.9
+categories          devel
+platforms           darwin
+license             MIT
+
+maintainers         {darkcog.com:casr @casr} openmaintainer
+description         A better fuzzy finder
+long_description    fzy tries to find the result the user intended. It does \
+                    this by favouring matches on consecutive letters and \
+                    starts of words. This allows matching using acronyms or \
+                    different parts of the path.
+
+checksums           sha256 f5e122251d41527007c0675276f414534337b5814d1bb6d23587f7a3dcc91de4 \
+                    rmd160 e278ef08985c476c7c70c6f870ea301abaecc6c9
+
+use_configure       no
```

Comment:
When setting `use_configure no`, this usually breaks universal builds. If you want to fix universal building for this port, you'll likely have to add the flags returned by `[get_universal_archflags cc]`/`[get_universal_archflags ld]` to your `CFLAGS`/`LDFLAGS`.

See https://trac.macports.org/wiki/UniversalDevelopment#Approaches for details. Alternatively, you can also disable the universal variant by setting `universal_variant no`.